kentikapi/client.go

```diff
@@ -150,3 +173,59 @@ func makeRetryingClientConfig(c Config) httputil.ClientConfig {
 		RetryCfg: c.RetryCfg,
 	}
 }
+
+//nolint: gosec
+func makeConnForSynthetics(c Config) grpc.ClientConnInterface {
+	if c.SyntheticsGRPCAPIURL == "" {
```

Comment:
IMO grpc.Dial() call should not depend on SyntheticsGRPCAPIURL like that, because user has no ability to create client with TLS enabled with custom URL. Please set default URL (hostPort) in NewClient() function and remove that if statement here.

If we need option to disable server cert verification for tests, this could be other config flag.

Reply:
Done